lib/main.rb

```diff
@@ -0,0 +1,17 @@
+class LogParser
+    def initialize(file_name)
+        @file_name = file_name
+    end
+
+    def print_first_line
+        begin
+            file = File.open(@file_name)
+            file_data = file.readlines.map(&:chomp)
+            first_line = file_data[0]
+            file.close
+            puts first_line
```

Comment:
If you just want to read the first line, you could do just do this:

```suggestion
            first_line = File.open(@file_name, &:readline)
            puts first_line
```

From the ruby [docs](https://rubydoc.info/stdlib/core/File#open-class_method):
> With no associated block, open is a synonym for File.new. If the optional code block is given, it will be passed file as an argument, and the File object will automatically be closed when the block terminates. In this instance, File.open returns the value of the block.